invenio_communities/members/records/mappings/os-v1/communitymembers/members/member-v1.0.0.json

```diff
@@ -68,6 +68,14 @@
               },
               "email_visibility": {
                 "type": "keyword"
+              },
```

Comment:
could you add the all the modified mappings' names to the "updated mappings" section in https://github.com/inveniosoftware/docs-invenio-rdm/issues/536 ? 
We will need to remember to include them in a version upgrade guide

Reply:
Could you add all the mappings names to the list please? I saw you added the modification of one only, we need the names of all the affected indices, archivedinvitation-v1.0.0, member-v1.0.0, etc. Please make sure you also checked other PRs for similar mappings upgrades and included them in the changelog issue

Reply:
Can you take a look and let me know if it looks good?